backend/maelstro/config/config.py

```diff
@@ -77,6 +77,30 @@ def has_db_logging(self) -> bool:
     def get_db_config(self) -> DbConfig:
         return DbConfig(**self.config.get("db_logging", {}))
 
+    def get_transformations(self) -> dict[str, Any]:
+        return self.config.get("transformations", {})  # type: ignore
+
+    def get_transformation_pair(self, src: str, dst: str) -> list[dict[str, Any]]:
+        return self.get_transformation_pairs()[f"{src} -> {dst}"]  # type: ignore
+
+    def get_transformation_pairs(self) -> dict[str, Any]:
+        transformations = {}
+        for src in self.config["sources"]["geonetwork_instances"]:
+            for dst_name, dst in self.config["destinations"].items():
+                current_transformations = [
+                    self.get_transformations().get(k)
+                    for k in src.get("transformations", [])
+                ]
+                current_transformations += [
+                    self.get_transformations().get(k)
+                    for k in dst.get("transformations", [])
+                ]
+                if current_transformations:
```

Comment:
Seems you will get a keyError if there is no transformations, why not setting an empty array.

Reply:
fix by default value